tests/h/streamer/metrics_test.py

```diff
@@ -0,0 +1,61 @@
+from unittest.mock import create_autospec
+
+import pytest
+from h_matchers import Any
+
+from h.streamer.metrics import websocket_metrics
+from h.streamer.websocket import WebSocket
+
+
+class TestWebsocketMetrics:
+    def test_it_records_socket_metrics(self, generate_metrics, sockets):
+        sockets[0].authenticated_userid = "acct:jimsmith@hypothes.is"
+        sockets[1].authenticated_userid = None
+        sockets[2].authenticated_userid = None
```

Comment:
This is technically redundant as we setup the None below, but I think it nicely illustrates why we get the numbers we do without having to look outside of the test function